### PR TITLE
Handling units for Nexus Aquifer methods

### PR DESCRIPTION
**Pull request by obi-isebor**

### CODE REVIEW

File: ResSimpy/Nexus/NexusSimulator.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def __init__(self, origin: Optional[str] = None, destination: Optional[str] = No
         self._rock: NexusRockMethods = NexusRockMethods()
         self._relperm: NexusRelPermMethods = NexusRelPermMethods()
         self._valve: NexusValveMethods = NexusValveMethods()
-        self._aquifer: NexusAquiferMethods = NexusAquiferMethods()
+        self._aquifer: NexusAquiferMethods = NexusAquiferMethods(model_unit_system=UnitSystem.ENGLISH)
```
Comment (jonnyh404) [inline ResSimpy/Nexus/NexusSimulator.py:116]: Is this just a temporary placeholder until it gets replaced by the below read in of properties?

File: ResSimpy/Units/AttributeMappings/DynamicPropertyUnitMapping.py [hunk 2]
```diff
@@ -3,9 +3,124 @@
...
+                                  )
+
+
+class AquiferUnits(BaseUnitMapping):
+    """Unit types for the attributes of Aquifer methods."""
+
+    def __init__(self, unit_system: None | UnitSystem) -> None:
+        super().__init__(unit_system=unit_system)
+
+    attribute_map: Mapping[str, UnitDimension] = {
```
Comment (jonnyh404) [inline ResSimpy/Units/AttributeMappings/DynamicPropertyUnitMapping.py:20]: Looks good to me

File: ResSimpy/Units/Units.py [hunk 3]
```diff
@@ -391,6 +391,16 @@ class Time(UnitDimension):
     metric_atm = 'days'
 
 
+class InverseTime(UnitDimension):
```
Comment (jonnyh404) [inline ResSimpy/Units/Units.py:394]: good being very explicit with these units.

**Review by jonnyh404 — APPROVED** [review]
Good test coverage, looks good to me!

File: ResSimpy/Nexus/NexusSimulator.py (hunk 1 above)
Comment (obi-isebor) [reply ResSimpy/Nexus/NexusSimulator.py:116]: @jonnyh404, yes, that's the idea. Open to suggestions though if you think there is a better way to do this

**Review by cflynn3 — APPROVED** [review]
Looks good.

**Pull request merged**